backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/AddVmCommand.java

```diff
@@ -843,6 +840,12 @@ protected boolean validate() {
         return true;
     }
 
+    private Map<Guid, List<DiskImage>> initStorageToDisksMap() {
```

Comment:
Not a best name, since it doesn't initialize `storageToDiskMap` indeed. But I understand your concerns, so let's leave it this way.

Reply:
Will follow Pavel's suggestion, thanks